client/app/cloud/project/storage/storage-add/storage-add.html

```diff
@@ -1,39 +1,37 @@
-<div class="cui-page">
+<section class="cui-page__content">
 
-    <div data-ng-include="'app/cloud/project/storage/storage-header.html'"></div>
+    <cui-message-container data-messages="messages"></cui-message-container>
 
-    <section class="cui-page__content">
-        <oui-back-button data-title="{{ :: contentTitle }}"></oui-back-button>
+    <oui-back-button data-title="{{ :: contentTitle }}"></oui-back-button>
 
-        <div data-ng-show="!loaders.init">
+    <div data-ng-show="!loaders.init">
 
-            <form role="form" name="addForm" novalidate="novalidate">
+        <form role="form" name="addForm" novalidate="novalidate">
 
-                <div class="row">
+            <div class="row">
 
-                    <div class="col-md-6 col-lg-4 center add-button-item"
-                         data-ng-repeat="item in items.value"
-                         data-ng-class="{
-                            'active': addedType.value == item
-                         }">
-
-                        <div class="form-group add-button-container add-button-container-{{item}}"
-                            data-ng-class="{
-                                'active': addedType.value == item
-                            }">
-                        </div>
+                <div class="col-md-6 col-lg-4 center add-button-item"
+                     data-ng-repeat="item in items.value"
+                     data-ng-class="{
+                        'active': addedType.value == item
+                     }">
 
-                        <a class="btn add-button" data-ng-class="{
-                            'active' : addedType.value == item
-                        }">{{ 'add_' + item | translate}}</a>
+                    <div class="form-group add-button-container add-button-container-{{item}}"
+                        data-ng-class="{
+                            'active': addedType.value == item
+                        }">
                     </div>
+
+                    <a class="btn add-button" data-ng-class="{
```

Comment:
It's a link, a button ? What does this do ?

Reply:
Honnestly, I don't have any idea of what does this link do. When displaying the page, there is... nothing.
Digging into the controller, I don't find any references of `items.value` (that is repeated above). So I think the `ngRepeat` is useless...